src/Components/Header/Header.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { FaMicrophone } from 'react-icons/fa';
+import { IoIosSettings, IoIosArrowBack } from 'react-icons/io';
+import header from './Header.module.css';
+import { displayCountryMap } from '../../redux/Pollution/pollution.js';
+
+const Header = () => {
+  const SetDisplay = () => {
+    useNavigate(-1);
+    useDispatch(displayCountryMap(false));
+  };
```

Comment:
- [ ] [OPTIONAL] I am making it as optional because it was not mentioned by the previous reviewer, however, I strongly suggest you fix this as this is one of the core requirements, currently, I am not able to go back from the country details page, it gives this error please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/235308277-f263e2fb-e1d7-4ce7-8755-a5b7b6c25ea0.png)

Now this is happening because you are calling a hook inside a regular function which is not allowed in React and will result in an error like the above. Hooks can only be called inside functional components or other custom hooks.

now how we fix this is simple, you call the hooks at the top and use it like :point_down: 
```
  const navigate = useNavigate();
  const dispatch = useDispatch();
  
  const SetDisplay = () => {
    navigate(-1);
    dispatch(displayCountryMap(false));
  };
```

Reply:
Hey @PinkMoon25 Done 👌